101-azurefirewall-premium/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't see anything that suggests the prereqs need to be in the same resource group - so you can remove this file.   you'll need to sure your resource references include the resourceGroup as well.